### DISCUSSION TITLE
The INSPIRE validator says the Schema is invalid but it is not.

### DISCUSSION THREAD

**Discussion by Jonazabal** [INSPIRE-MIF/helpdesk-validator#833] [Q&A] [answered]
We are using Map Server for Windows for our WMS services, we have tested that the GetCapabilities XML validates against the XML-Schema and it does but according to the INSPIRE validator it is not valid. Our services has not been visible in the EU-portal since 1.5 years and we do not know how to proceed. 

https://geodata.socialstyrelsen.se/cgi/mapserv.exe?map=/ms4w/maps/sos-dod-wms.map&service=wms&request=GetCapabilities

Error:
Schema not valid: [org.xml.sax.SAXException: Fatal error: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 7; cvc-elt.1: Cannot find the declaration of element 'HTML'. Response did not validate against schema 'http://inspire.ec.europa.eu/schemas/inspire_vs/1.0/inspire_vs.xsd'.]

**Comment by fabiovinci** [comment, marked as answer]
Dear @Jonazabal,

the GetCapabilities response of your WMS seems to be valid if validated against the INSPIRE VS schema (http://inspire.ec.europa.eu/schemas/inspire_vs/1.0/inspire_vs.xsd).

I will convert the discussion into an issue and we will investigate further.
